news-bundle/src/Resources/contao/modules/ModuleNews.php

```diff
@@ -387,6 +407,16 @@ protected function generateLink($strLink, $objArticle, $blnAddArchive=false, $bl
 			($blnIsReadMore && $blnIsInternal ? '<span class="invisible"> ' . $objArticle->headline . '</span>' : '')
 		);
 	}
+
+	private function getNewsContent(array $options = [])
+	{
+
+	}
+
+	private function hasNewsContent()
+	{
+
+	}
```

Comment:
Is this maybe a leftover?

Reply:
hah yes! I've moved that to static methods in `News.php` to make it available for RSS etc.